app/controllers/admin/sessions_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Admin::SessionsController < Admin::AdminController
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.